install/kubernetes/helm/subcharts/istiocoredns/coredns.yaml

```diff
@@ -0,0 +1,125 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: istio-coredns-plugin
+  namespace: istio-system
+---
+# Source: istio/charts/pilot/templates/clusterrole.yaml
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: istio-coredns-plugin
+  namespace: istio-system
+rules:
+- apiGroups: ["networking.istio.io"]
+  resources: ["*"]
+  verbs: ["get", "watch", "list"]
+---
+# Source: istio/charts/pilot/templates/clusterrolebinding.yaml
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: istio-coredns-plugin
+  namespace: istio-system
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: istio-coredns-plugin
+subjects:
+  - kind: ServiceAccount
+    name: istio-coredns-plugin
+    namespace: istio-system
```

Comment:
hard-coded namespace, maybe it should be replaced with `{{ .Release.Namespace }}`